app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
   <head>
-    <title>Early careers framework</title>
+    <title><%= (yield(:title) + " - " unless yield(:title).blank?).to_s + "Early career framework" %></title>
```

Comment:
I think if you interpolate yield(:title) + " - " you do away with the need for to_s

Reply:
@amazimbe I think otherwise when :title is blank, you get `nil` and can't do `nil + string`